src/mme-app/sec/secUtils.cpp

```diff
@@ -114,7 +114,7 @@ void SecUtils::create_nh_key(const unsigned char *kasme, unsigned char *nh_key,
 {
        uint8_t salt[35] = {0};
        salt[0] = NEXT_HOP_FC; /*TODO : Sec algo. handle properly instead of harcoding here*/
```

Comment:
Impact of this comment ?

Reply:
this value is constant for 4g. Don't think it has an impact.